modules/imaging_uploader/js/imaging_uploader.js

```diff
@@ -228,20 +229,22 @@ function UploadProgress() {
             }
         }
                 
+        // Display pSCID, CandID and VisitLabel for the selected upload
+        var progressHeader = progressType + ' of upload ' + uploadId + ' for ' + pscid + ' (CandID ' + candid + ') at ' + visitLabel + "\n";
+        
         // If pipeline is still running
         if(this.isInserting()) {
-            return progressType + ' of upload ' + uploadId + ' for ' + pscid + ' at ' + visitLabel + "\n"
+            return progressHeader
             + notificationText
             + this._dots
             + ['|', '/', '-', '\\', '|', '/', '-', '\\'][this._animatedCharIndex % 8] + "\n";
         }
         
         // Pipeline finshed running: failed or succeeded ?
-        var statusText = 'Upload is finished and ' + (this.isInsertionComplete() ? 'was successful' : 'failed');
+        var statusText;
+        if() = 'Upload is finished and ' + (this.isInsertionComplete() ? 'was successful' : 'failed');
```

Comment:
@nicolasbrossard, 

var statusText = 'Upload is ...' (remove the if() statement)

then it will be Okay when testing it.